test/integration/emitter_singleton_test.ts

```diff
@@ -12,36 +12,36 @@ import { fixture, assertStructured } from "./emitter_factory_test";
 import { rejects, doesNotReject } from "assert";
 
 describe("Emitter Singleton", () => {
```

Comment:
It should be a goal to not require parameterization for this file. Can we try adding this feature without a change to this file?

A separate test for parameterized messages would be fine if needed.